includes/DomainCheckJob.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Miraheze\RequestSSL;
```

Comment:
This should be moved to includes/Jobs/DomainCheckJob and use `namespace Miraheze\RequestSSL\Jobs;` IMO anyway

Reply:
Done in [a679c6b](https://github.com/miraheze/RequestSSL/pull/40/commits/a679c6b9d571b0278ce7e3941aa10aded3ffff03)